azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/utils/Utils.java

```diff
@@ -157,6 +150,40 @@ public static boolean isExecutableJar(File file) {
         }
     }
 
+    public static <T> T executeCallableWithPrompt(Callable<T> callable, String prompt, int timeOutInSeconds) {
+        final ExecutorService executorService = Executors.newFixedThreadPool(2);
+        final Future<T> future = executorService.submit(callable);
+        final Future<T> wrappedFuture = executorService.submit(getWrappedCallable(prompt, future));
+        try {
+            return wrappedFuture.get(timeOutInSeconds, TimeUnit.SECONDS);
+        } catch (Exception e) {
+            wrappedFuture.cancel(true);
+            return null;
+        } finally {
+            executorService.shutdown();
+        }
+    }
+
+    private static List<File> getArtifacts(List<Resource> resources) {
+        final List<File> result = new ArrayList<>();
+        final DirectoryScanner directoryScanner = new DirectoryScanner();
+        for (final Resource resource : resources) {
+            if (resource.getIncludes() != null && resource.getIncludes().size() > 0) {
+                directoryScanner.setBasedir(resource.getDirectory());
```

Comment:
I think the following utils are doing the same thing
 org.apache.commons.io.filefilter.FileFilterUtils;
 org.apache.commons.io.filefilter.IOFileFilter;
 org.apache.commons.io.filefilter.NameFileFilter;
 org.apache.commons.io.filefilter.WildcardFileFilter;

Reply:
This will be done in following pr.